lib/_io.scm

```diff
@@ -8343,6 +8343,167 @@
         (macro-dynamic-bind output-port port thunk)))))
 
 ;;;----------------------------------------------------------------------------
+(define-prim (##open-raw-device
```

Comment:
dunno, I kind of like the term `raw` -- it implies `unmanaged`. 
But up to you, although I would prefer to not have to change the name in too many places (ie in os_io).